SU2_CFD/src/fluid/CSU2TCLib.cpp

```diff
@@ -1817,10 +1877,58 @@ void CSU2TCLib::ThermalConductivitiesWBE(){
   ThermalConductivities[1] = ThermalCond_ve;
 }
 
-void CSU2TCLib::DiffusionCoeffGY(){
+su2double CSU2TCLib::ComputeCollisionCrossSection(unsigned iSpecies, unsigned jSpecies, su2double T, bool d1, bool coulomb) {
+
+  const su2double pi = PI_NUMBER;
+  const su2double Na = AVOGAD_CONSTANT;
+  
+  if (coulomb) {
+    
+    const su2double e_cgs = FUND_ELEC_CHARGE_CGS; // CGS unit of fundamental electric charge 
+    const su2double kb_cgs = BOLTZMANN_CONSTANT * 1E7; // CGS unit of Boltzmann Constant 
+    const su2double ne_cgs = Na * rhos[0] / MolarMass[0] * 1E-6; // CGS unit of electron number density
+        
+    const su2double debyeLength = sqrt(kb_cgs * T / 4 / pi / ne_cgs / pow(e_cgs,2));
+    const su2double T_star = debyeLength / (pow(e_cgs,2) / (kb_cgs * T));
+
+    if (Omega11(iSpecies, jSpecies, 0) == 1.0 && d1) {
```

Comment:
Can we add some comments to this block just for clarity?

Reply:
Sure, but I think this is best explained by the PR to documentation which gives the actual equations.

Reply:
I think something like  'Compute collision term'  and something to help distinguish the loops a little bit.  Nothing crazy.